resources/Microsoft.PackageManagement.NuGetProvider.Resources.Messages.resx

```diff
@@ -0,0 +1,385 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@alerickson this file is auto-generated. Please do not check it in.

Reply:
I'll remove it